src/x11/screen.c

```diff
@@ -98,7 +98,8 @@ void randr_init()
 
 void randr_update()
 {
-        if (randr_minor_version < 5) {
+	if (randr_major_version > 1
```

Comment:
Noticed this just as I was about to click merge:

We're checking for the error condition so it should be `randr_major_version < 1`